spec/jobs/in_person/enrollments_ready_for_status_check/enrollment_pipeline_spec.rb

```diff
@@ -0,0 +1,334 @@
+require 'rails_helper'
+
+RSpec.describe InPerson::EnrollmentsReadyForStatusCheck::EnrollmentPipeline do
+  let(:error_reporter) { instance_double(InPerson::EnrollmentsReadyForStatusCheck::ErrorReporter) }
+  let(:email_body_pattern) { /\A\s*(?<enrollment_code>\d{16})\s*\Z/ }
```

Comment:
Do we want to grab this from the config so if it changes there the test will be aware?

Reply:
This is a unit test; testing the specific configuration is out-of-scope. This is meant to test that the module functions correctly as a unit.